src/tests/complex_tests/mod.rs

```diff
@@ -176,7 +176,7 @@ pub(crate) fn generate_base_layer(
     use crate::external_calls::run;
     use crate::toolset::GeometryConfig;
 
-    let mut storage_impl = InMemoryStorage::new();
+    let mut storage_impl = InMemoryCustomRefundStorage::new(None);
```

Comment:
Do we really need to change it for `complex_test`?

Reply:
only because the underlying function `save_predeployed_contracts` changed. overall its the same implementation as what was used before when `None` is passed in. It made more sense for this to live in this repo vs `zk_evm`

Reply:
Since we need only change `get_access_refund` implementation, may be it would be better to create kind of a wrapper over `InMemoryStorage`? Otherwise, over time, something in `InMemoryStorage` may change, and we will forget to copy it to `InMemoryCustomRefundStorage`

Reply:
I thought about that but given both are only used for tests I don't think it makes a significant difference. I don't feel strongly one way or the other though

Reply:
I would prefer the approach of creating a wrapper around imported `InMemoryStorage`. Then, in any case, we will have less code in the repository and it will be easier to maintain consistency

Reply:
Also we now have a lot of new compile warnings because visibility of `utils` changed. I think that `storage` do not belong to `utils` since it is used somewhere in `complex_tests`